tracer/src/Datadog.Trace/Iast/Aspects/Newtonsoft.Json/NewtonsoftJsonAspects.cs

```diff
@@ -0,0 +1,208 @@
+// <copyright file="NewtonsoftJsonAspects.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Iast.Dataflow;
+using Datadog.Trace.Logging;
+
+#nullable enable
+
+namespace Datadog.Trace.Iast.Aspects.Newtonsoft.Json;
+
+/// <summary> Newtonsoft.Json class aspects </summary>
+[AspectClass("Newtonsoft.Json")]
+[global::System.ComponentModel.Browsable(false)]
+[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+public class NewtonsoftJsonAspects
+{
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<NewtonsoftJsonAspects>();
+
+    private static readonly Type JObjectType = Type.GetType("Newtonsoft.Json.Linq.JObject, Newtonsoft.Json")!;
+    private static readonly Type JArrayType = Type.GetType("Newtonsoft.Json.Linq.JArray, Newtonsoft.Json")!;
+    private static readonly Type JTokenType = Type.GetType("Newtonsoft.Json.Linq.JToken, Newtonsoft.Json")!;
```

Comment:
I'd put this initializations in a static constructor in a try catch, to be able to log a warning in case something goes south